### PR TITLE
OCPBUGSM-23132 - Allow read-only-admin to call openshift_versions call

### PR DESCRIPTION
**Pull request by eliorerz**
[OCPBUGSM-23132](https://issues.redhat.com/browse/OCPBUGSM-23132) - Add listSupportedOpenshiftVersions test.

### CODE REVIEW

File: pkg/auth/authz_handler_test.go [hunk 1]
```diff
@@ -463,6 +463,11 @@ var _ = Describe("authz", func() {
 			allowedRoles: []ocm.RoleType{ocm.AdminRole, ocm.ReadOnlyAdminRole, ocm.UserRole},
 			apiCall:      listComponentVersions,
 		},
+		{
+			name:         "list supported openshift versions",
```
Comment (filanov) [inline pkg/auth/authz_handler_test.go:467]: I don't think that a test here will help here, looks like this validations is not related directly the same code that run with swagger, so the permission here is mocked, if you set the permissions to be `allowedRoles` then this is how the mocks will be defined, if you want to validate the change in the swagger, i needs to be done in the subsystem.
@rollandf correct me if i'm wrong.

Comment (rollandf) [reply pkg/auth/authz_handler_test.go:467]: The test fails if run before the swagger change, so it looks it does it OK.

Comment (rollandf) [reply pkg/auth/authz_handler_test.go:467]: The test fails if run before the swagger change, so it looks it does it OK.

Comment (filanov) [reply pkg/auth/authz_handler_test.go:467]: it compare to the validation in the swagger? because i see that it mock the capability `passCapabilityReview`

Comment (filanov) [reply pkg/auth/authz_handler_test.go:467]: ok i see it. thanks

**Review by filanov — APPROVED** [review]
